dev/lis-gene-search-element.html

```diff
@@ -31,12 +31,15 @@ <h1>&lt;lis-gene-search-element&gt;</h1>
         This allows users to share specific pages from a search via the URL and for the search history to be navigated via the Web browser's forward and back buttons.
         If the query string parameters are present when the component loads then a search will be automatically performed with the query string parameter values.
       </p>
+      <p>
+        A search for a specific genus can be enabled by specifying the <code>genus</code> attribute in the HTML tag.
```

Comment:
Rephrase "A search for a specific genus..." to "A search can be restrict to a specific genus..."